manifest.toml

```diff
@@ -0,0 +1,104 @@
+packaging_format = 2
+
+id = "mattermost"
+name = "Mattermost"
+description.en = "Open source collaboration platform built for developers"
+description.fr = "Plateforme de collaboration open source conçue pour les développeurs"
+
+version = "7.9.1~ynh1"
+
+maintainers = ["pmorinerie"]
+
+[upstream]
+license = "GPL-3.0-only"
+website = "http://www.mattermost.org/"
+admindoc = "https://docs.mattermost.com/guides/deployment.html"
+userdoc = "https://docs.mattermost.com/guides/messaging.html"
+code = "https://github.com/mattermost/mattermost-server"
+
+[integration]
+yunohost = ">= 11.1.15"
+architectures = ["amd64", "armhf", "arm64"]
+multi_instance = true
+ldap = false
+sso = false
+disk = "50M"
+ram.build = "350M"
+ram.runtime = "50M"
+
+[install]
+    [install.domain]
+    type = "domain"
+
+    [install.path]
+    type = "path"
+    default = "/mattermost"
+
+    [install.init_main_permission]
+    help.en = "If enabled, Mattermost will be accessible by the user without a YunoHost account. This can be changed later via the webadmin."
```

Comment:
```suggestion
    help.en = "If enabled, Mattermost will be accessible by users without a YunoHost account. This can be changed later via the webadmin."
```